src/net_processing.cpp

```diff
@@ -301,6 +370,9 @@ struct CNodeState {
     }
 };
 
+// Keeps track of the time (in microseconds) when transactions were requested last time
+limitedmap<uint256, int64_t> m_already_asked_for GUARDED_BY(cs_main)(MAX_INV_SZ);
```

Comment:
nit: I mentioned this before, I believe our style guide says that globals should start with `g_` instead of `m_`.